lib/mutant/ast/meta/resbody.rb

```diff
@@ -0,0 +1,15 @@
+module Mutant
+  module AST
+    # Node meta information mixin
+    module Meta
+
+      # Metadata for resbody nods
```

Comment:
I think "resbody nods" is a typo and should be "resbody nodes" (the former text is already present in the current HEAD)?